pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[tool.poetry]
+name = "rexfw"
+version = "0.1.0"
+description = "Replica Exchange for computing clusters"
+authors = ["simeoncarstens <simeon.carstens@tweag.io>"]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+numpy = "*"
+mpi4py = "*"
+pyyaml = "*"
+# we have to wait
+# for https://github.com/python-poetry/poetry/pull/2242
+# set RESAAS_COMMON_PATH instead
+# resaas-common = { path = "../../resaas/common/"  }
```

Comment:
We have this weird setup at the moment where resaas is a monorepo on which rexfw also depends. In principle we should probably move rexfw to the resaas repo if we stick with the monorepo approach, but for the time being I think we might be able to get away with a git submodule pointing at the resaas repo (so this would be a path dependency kind of like in the comment)

Reply:
Added a `resaas` submodule.